app/src/main/res/layout/fragment_input_hostname.xml

```diff
@@ -49,4 +49,13 @@
                 app:fabSize="mini"
                 app:srcCompat="@drawable/ic_arrow_forward_white_24dp" />
     </LinearLayout>
+
+    <TextView
+            android:id="@+id/version_info"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal|bottom"
+            android:layout_margin="@dimen/margin_8"
+            android:textColor="@android:color/white"
+            android:textAppearance="?attr/textAppearanceListItemSmall" />
```

Comment:
This text is not a list item, so it is better to use `@style/TextAppearance.AppCompat.Caption` or `@style/TextAppearance.AppCompat.Body1` instead.